tests/FSharpx.Collections.Experimental.Tests/AaTreeTest.fs

```diff
@@ -0,0 +1,129 @@
+namespace FSharpx.Collections.Experimental.Tests
+
+open FSharpx.Collections
+open FSharpx.Collections.Experimental
+open Expecto
+open Expecto.Flip
```

Comment:
I believe that you are not using the flip style and that the code will compile again if you remove it.